scenarios/grpc_search.js

```diff
@@ -0,0 +1,32 @@
+import { check } from 'k6';
+import native from 'k6/x/neofs/native';
+
+const grpc_client = native.connect(__ENV.GRPC_ENDPOINT, '', 5, 15);
```

Comment:
id make consts for non-obvious literals

Reply:
dont mind

Reply:
and empty string pls, i can only imagine what does it mean